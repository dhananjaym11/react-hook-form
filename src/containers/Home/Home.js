import React from 'react';

import './Home.scss';
import HomeForm from '../../components/HomeForm/HomeForm';
import HomeFormOne from '../../components/HomeForm/HomeFormOne';
import HomeFormTwo from '../../components/HomeForm/HomeFormTwo';
import HomeFormThree from '../../components/HomeForm/HomeFormThree';

class HomeContainer extends React.Component {

    render() {
        return (
            <div className="home-page">
                Home page

                {/* <hr />
                <HomeForm />
                <hr />
                <HomeFormOne />
                <hr />
                <HomeFormTwo /> */}
                <hr />
                <HomeFormThree />
            </div>
        )
    }
}

export default HomeContainer;