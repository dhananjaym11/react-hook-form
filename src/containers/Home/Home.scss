.home-page {
    .msg {
        font-size: 1.2rem;
        padding: 5px 7px;
        margin: 0;
    }
}