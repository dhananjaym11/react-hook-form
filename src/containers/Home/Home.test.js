import { shallow } from 'enzyme';
import React from 'react';

import Home from './Home';

it('Render component', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
});