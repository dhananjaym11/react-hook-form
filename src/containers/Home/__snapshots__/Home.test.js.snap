// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component 1`] = `ShallowWrapper {}`;
